gcp-bundle/NOTICE

```diff
@@ -29,72 +29,225 @@ from the source code management (SCM) system project uses.
 
 --------------------------------------------------------------------------------
 
-NOTICE for Group: commons-codec  Name: commons-codec  Version: 1.15
+NOTICE for Group: commons-codec  Name: commons-codec  Version: 1.17.1
 
-src/test/org/apache/commons/codec/language/DoubleMetaphoneTest.java
-contains test data from http://aspell.net/test/orig/batch0.tab.
-Copyright (C) 2002 Kevin Atkinson (kevina@gnu.org)
-
-===============================================================================
-
-The content of package org.apache.commons.codec.language.bm has been translated
-from the original php source code available at http://stevemorse.org/phoneticinfo.htm
-with permission from the original authors.
-Original source copyright:
-Copyright (c) 2008 Alexander Beider & Stephen P. Morse.
+| Apache Commons Codec
+| Copyright 2002-2024 The Apache Software Foundation
+| 
+| This product includes software developed at
+| The Apache Software Foundation (https://www.apache.org/).
```

Comment:
Minor: I'd prefer not to include boilerplate from ASF since it isn't required.

Reply:
Agree (I wanted to be consistent with other bundle/runtime jar). Let me improve that 😄

Reply:
Done !